collections/announcements/2025-03-24-reflections-restoration-progress-restore-council-2025.md

```diff
@@ -6,7 +6,7 @@ date: 2025-03-24
 
 The Gulf Coast Ecosystem Restoration Council (RESTORE Council) is commemorating 15-years since  the Deepwater Horizon oil spill with a month-long reflection on its progress to date implementing meaningful ecosystem and economic restoration across the Gulf Coast. To date, the RESTORE Council has invested over $1.1 billion in Gulf Coast restoration. Over $470 million has been awarded through the Council-Selected Restoration Component for Gulfwide ecosystem restoration and protection. In that same period, over $679 million has been awarded through the Spill Impact Component to contribute to the overall economic and ecological recovery of the Gulf coast region.  Over its lifetime, the RESTORE Council will invest over $3 billion in restoration activities in the Gulf Coast, advancing the RESTORE Council’s vision:
 
-![RESTORE Council Vision Statement](/img/eblast_image_03242025.jpg)
+![RESTORE Council Vision Statement - A visual representation of the council's goals and objectives](/img/eblast_image_03242025.jpg)
```

Comment:
Since the actual vision statement isn't in text anywhere, just on the image, let's make the alt text the complete vision statement